qiskit/circuit/library/standard_gates/x.py

```diff
@@ -1280,6 +1268,17 @@ def __new__(
             return gate
         return super().__new__(cls)
 
+    @deprecate_func(
+        additional_msg=(
+            "It is recommended to use :class:`.MCXGate` and let :class:`.HighLevelSynthesis` choose "
+            "the best synthesis method depending on the number of ancilla qubits available. "
+            "If this specific synthesis method is required, one can specify it using the plugin "
+            "interface. Alternatively, one can use :func:`.synth_mcx_gray_code` to construct "
```

Comment:
Do you want to point out the name of the plugin option? Otherwise we could add an example in the release notes.

Reply:
Good point, I have done this.